static/app/components/dropdownAutoComplete/list.tsx

```diff
@@ -80,15 +86,13 @@ const List = ({
                 ? virtualizedLabelHeight
                 : virtualizedHeight
             }
-            rowRenderer={({key, index, style}: ListRowProps) => (
+            rowRenderer={({key, index, style}) => (
               <Row
                 key={key}
-                item={items[index]}
                 style={style}
-                itemSize={itemSize}
-                highlightedIndex={highlightedIndex}
-                inputValue={inputValue}
-                getItemProps={getItemProps}
+                item={items[index]}
+                isActive={items[index].index === highlightedIndex}
+                {...rowProps}
```

Comment:
Unfortunately I wasn't able to give quite the same performance boost to rows in the react-virutalized list, since we pass the `style` prop in, meaning all rows need to get re-rendered anyway :(

Reply:
Oh maybe. I think I it also repositions the object esch time too